internal/tests/pytests/wrappers/pair_stat/test_pair_stat_wrapper.py

```diff
@@ -0,0 +1,756 @@
+#!/usr/bin/env python3
+
+import pytest
+
+import os
+
+from metplus.wrappers.pair_stat_wrapper import PairStatWrapper
+
+pairs_dir = '/some/path/pairs'
+pair_stat_format = 'mpr'
+
+def set_minimum_config_settings(config, fcst_and_obs_data):
+    _, _, _, _, inits, _, lead_hour, time_fmt = fcst_and_obs_data
+    # set config variables to prevent command from running and bypass check
+    # if input files actually exist
+    config.set('config', 'DO_NOT_RUN_EXE', True)
+    config.set('config', 'INPUT_MUST_EXIST', False)
+
+    # set process and time config variables
+    config.set('config', 'PROCESS_LIST', 'PairStat')
+    config.set('config', 'LOOP_BY', 'INIT')
+    config.set('config', 'INIT_TIME_FMT', time_fmt)
+    config.set('config', 'INIT_BEG', inits[0])
+    config.set('config', 'INIT_END', inits[-1])
+    config.set('config', 'INIT_INCREMENT', '12H')
+    config.set('config', 'LEAD_SEQ', f'{lead_hour}H')
+
+    config.set('config', 'PAIR_STAT_FORMAT', pair_stat_format)
+    config.set('config', 'PAIR_STAT_CONFIG_FILE',
+               '{PARM_BASE}/met_config/PairStatConfig_wrapped')
+    config.set('config', 'PAIR_STAT_PAIRS_INPUT_DIR', pairs_dir)
+    config.set('config', 'PAIR_STAT_PAIRS_INPUT_TEMPLATE',
+               '{init?fmt=%Y%m%d%H}/fcst_file_F{lead?fmt=%3H}')
+    config.set('config', 'PAIR_STAT_OUTPUT_DIR',
```

Comment:
Possible impact here related to Pair-Stat's use of `-out` on the command line rather than `-outdir`.